modules/stochastic_tools/doc/content/modules/stochastic_tools/examples/poly_chaos_surrogate.md

```diff
@@ -0,0 +1,215 @@
+# Polynomial Chaos Surrogate
+
+This example goes over how to train and utilize a [PolynomialChaos](PolynomialChaos.md) surrogate model. For detailed information on the polynomial chaos method including the mathematics and implementation see [PolynomialChaos](PolynomialChaos.md) and [QuadratureSampler](QuadratureSampler.md). For general information on training and evaluating a surrogate model see [Training a surrogate model](/examples/surrogate_training.md) and [Evaluating a surrogate model](/examples/surrogate_evaluate.md).
+
+## Overview
+
+Polynomial chaos (PC) is a stochastic element method. Its formulation is quite similar to finite element method, but instead of discretizing spatial dimensions, PC discretizes random variables, i.e. uncertain parameters. Within the training phase, PC creates a functional of a quantity of interest (QoI) that is dependent on these random variables. This function consists of a finite sum of orthogonal polynomials with coefficients, these coefficients are what the training phase is attempting to compute. Another difference from finite elements is that these polynomials do not necessarily have compact support and increasing the terms in the sum increases the polynomial order. Please see [PolynomialChaos.md] for more details of the polynomial chaos method.
```

Comment:
FYI. MooseDocs has an acronym extension that allows you to define PC in the config file and then use [!ac](PC) and it will automatically use the full name first and the acronym elsewhere on the page. It also shows the complete name if you hover over the acronym.